From 936da7fcab06ff3bc7c1c1e1ab108a36797da039 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 12 Aug 2017 09:28:00 -0700
Subject: [PATCH 3/4] initialize use_config_mr

Fixes
error: variable 'use_contig_mr' is uninitialized when used here [-Werror,-Wuninitialized]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 examples/dcini.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/dcini.c b/examples/dcini.c
index 001e905..08d75ec 100644
--- a/examples/dcini.c
+++ b/examples/dcini.c
@@ -269,7 +269,7 @@ int main(int argc, char *argv[])
 	int			size = 4096;
 	int			iters = 1000;
 	int			use_event = 0;
-	int			use_contig_mr;
+	int			use_contig_mr = 0;
 	int			err;
 	struct ibv_ah_attr	ah_attr;
 	struct dc_ctx		ctx = {
-- 
2.14.1

